### PR TITLE
sdformat8: update commit

### PR DESCRIPTION
**Pull request by chapulina**
Needed by [this ign-gazebo PR](https://bitbucket.org/ignitionrobotics/ign-gazebo/pull-requests/143/a-6-user-command-spawn-factory-create/diff)

### CODE REVIEW

File: Formula/sdformat8.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class Sdformat8 < Formula
   desc "Simulation Description Format"
   homepage "http://sdformat.org"
-  url "https://bitbucket.org/osrf/sdformat/get/6ae595df61ad1327cebbbce.tar.gz"
-  version "8.0.0~pre4~1~6ae595d"
-  sha256 "839edf3e4cee565c9642e9ffc23f041356ae992fdfc7f828c2e5571d999f81a4"
+  url "https://bitbucket.org/osrf/sdformat/get/301cecc99c227bad0b783477a2c4f7004b99961e.tar.gz"
+  version "8.0.0~pre4~1~301cecc9"
```
Comment (scpeters) [inline Formula/sdformat8.rb]: bump the `~1~` to `~2~`

**Comment by scpeters** [comment]
I'm not sure why this build failed:

* https://build.osrfoundation.org/job/generic-release-homebrew_bottle_builder/138/label=osx_mojave/console

it may be due to a bug in homebrew, but I'm not sure

* https://github.com/Homebrew/homebrew-test-bot/issues/231

File: Formula/sdformat8.rb (hunk 1 above)
Comment (chapulina) [reply Formula/sdformat8.rb]: Bumped in https://github.com/osrf/homebrew-simulation/pull/583/commits/6af391f7f7dcb1f212a463c786f40264eccf8815 and also removed msgs to #584 so we can tackle one lib at a time.

**Comment by chapulina** [comment]
> it may be due to a bug in homebrew, but I'm not sure

Will this prevent this PR from being merged?

**Comment by scpeters** [comment]
the bottle built, so the homebrew bug must not be a problem anymore

one of the travis builds is always 🔴 when the PR is from a branch of `osrf/homebrew-simulation`. it skips that build when building from a fork, so I just ignore the push build. this looks good to me.

**Pull request merged**